swap/swap_test.go

```diff
@@ -253,7 +257,7 @@ func TestRestoreBalanceFromStateStore(t *testing.T) {
 
 //create a test swap account
 //creates a stateStore for persistence and a Swap account
-func createTestSwap(t *testing.T) (*Swap, string) {
+func createTestSwap(t *testing.T, backend *backends.SimulatedBackend) (*Swap, string) {
```

Comment:
I think this would be more readable if there are two functions:
`createTestSwapWithBackend(t, backend)`, and `createTestSwap(t)` which calls the former with the default backend.

Then we wouldn't have to guess what all the `nil` arguments passed to the function are.

Reply:
shall we just use `new` instead of `create`

Reply:
Well there is a `New` for the actual "production" `swap.

But if you mean it to be `newTestSwap` I agree to this.